packages/commercetools/composables/src/factories/useInventoryFactory.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  ComputedProperty,
+  configureFactoryParams,
+  Context, CustomQuery,
+  FactoryParams,
+  Logger, PlatformApi,
+  sharedRef
+} from '@vue-storefront/core';
+
+export interface UseInventory<INVENTORY> {
+  entries: ComputedProperty<INVENTORY[]>;
+  inventory_entry: ComputedProperty<INVENTORY>;
+  load: (context: Context, params: { id: string } & { customQuery?: CustomQuery }) => Promise<void>;
```

Comment:
Should it return `void` in Promise?

Reply:
I don't see usage of `customQuery`. Where is it used? Also affects `search` types.